packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -116,7 +116,12 @@ export function logError(boundary: Fiber, errorInfo: CapturedValue<mixed>) {
     // A cycle may still occur if logCapturedError renders a component that throws.
     const suppressLogging = e && e.suppressReactErrorLogging;
     if (!suppressLogging) {
-      console.error(e);
+      // Rethrow it from a clean stack because this function is assumed to never throw.
+      // We can't safely call console.error() here because it could *also* throw if overriden.
+      // https://github.com/facebook/react/issues/13188
+      setTimeout(() => {
+        throw e;
+      });
```

Comment:
Since you set `suppressReactErrorLogging` to false, will this execute after the test above? Should you use `jest.useFakeTimers()` and force it to run with something like `jest.runAllTimers()`?

Reply:
I didn't really intend to test this part although I guess I could. I only wanted to ensure that we don't blow up the reconciler by rethrowing.

Reply:
Added a check.

Reply:
Cool, my primary concern was just confusion in test reporting if the error showed up randomly in the test suite because Node or Jest saw an uncaught error.

I just ran a quick test with:

```
  it(function() {
    setTimeout(() => {
      throw 'What';
    });
  });
```

No need to chase this down too much.

Reply:
Ah cool. Posted too late! I didn't realize that fake timers were already setup, so this wouldn't have run anyway.